logsetup.go

```diff
@@ -16,10 +17,10 @@ func SetupLogger(logLevel log.Level, fileLoggingEnabled bool) {
 	lumberjackLogger = &lumberjack.Logger{
 		// Log file absolute path, os agnostic
 		Filename:   filepath.ToSlash("logs/golbat.log"),
-		MaxSize:    50, // MB
-		MaxBackups: 10,
-		MaxAge:     30,   // days
-		Compress:   true, // disabled by default
+		MaxSize:    config.Config.Logging.MaxSize, // MB
```

Comment:
I wonder if we should do the config loading when calling `SetupLogger()` and add them as params or if we should keep it like that? @jfberry

Reply:
You can argue for a seperation of concerns where these parameters are chosen outside, and config is not imported to logsetup.